monkey/monkey_island/cc/ui/src/components/report-components/SecurityReport.js

```diff
@@ -241,6 +241,11 @@ class ReportPageComponent extends AuthComponent {
   generateReportGlanceSection() {
     let exploitPercentage =
       (100 * this.state.report.glance.exploited_cnt) / this.state.report.glance.scanned.length;
+
+    if (isNaN(exploitPercentage)) {
+      exploitPercentage = 0;
```

Comment:
0 of 0 is not 0%. Instead of displaying percentages we should simply not show the "percentage" section. I'm not sure what it adds anyway...